src/sagemaker/predictor.py

```diff
@@ -93,6 +93,12 @@ def predict(self, data):
         response_body.close()
         return data
 
+    def delete_endpoint(self):
+        def delete_endpoint(self):
+            """Delete an Amazon SageMaker ``Endpoint``.
```

Comment:
How about "Delete the Amazon SageMaker endpoint backing this predictor" or similar?

Reply:
Will do